appimagebuilder/app_dir/deploy/pacman/venv.py

```diff
@@ -170,10 +195,48 @@ def _generate_config(self):
                         f.write("Server = %s\n" % server)
 
     def _configure_keyring(self):
-        command = "pacman-key --config %s --init" % self._config_path
+        self._run_command("{sudo} {pacman_key} --config {config} --init")
+        self._run_command("{sudo} pacman-key --config {config} --populate archlinux")
+
+    def _run_command(self, command,
+                     stdout=sys.stdout,
+                     assert_success=True,
+                     wait_for_completion=True,
+                     wait_for_completion_timeout=6000,
```

Comment:
In some scenarios a command can take a lot too complete (like when the network connection is too slow).

It would be better to wait "forever" by default as there is no way of knowing the right times.

Reply:
Ok, I will replace `6000` with `None`, so it waits infinitely, if at all, at any time we require to provide a timeout, for example, in `appimagetool`, we can explicitly provide the keyworrd argument :+1:

Reply:
Sounds good!